begin/cmdline.py

```diff
@@ -37,10 +37,11 @@ def __init__(self, env_prefix=None, config_file=None, config_section=None):
         self._use_env = env_prefix is not None
         self._prefix = '' if not self._use_env else env_prefix
         self._parser = configparser.ConfigParser()
-        self._section = config_section
+        self._section = None
```

Comment:
Just trying to hide instance variables... ``DefaultsManager._section`` is for private instance variable, which should never be referenced directly. You should use ``DefaultsManager.section``, instead, which has getter and setter methods.